### PR TITLE
Feature/nconf

### PR DESCRIPTION
**Pull request by tdfischer**
+r

### CODE REVIEW

File: lib/config.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+var nconf = require('nconf');
+var dotenv = require('dotenv');
+
+dotenv.load();
+
+nconf.use('memory');
+nconf.env();
+
+nconf.defaults({
+  aws: {
```
Comment (stevenzeiler) [inline lib/config.js:10]: Why use the aws namespace? If you just make the key "AWS_ACCESS_KEY_ID" then it will be autoloaded from the environment?

Comment (stevenzeiler) [inline lib/config.js:10]: Ah I assume to make it prettier not having to type in ALL CAPS all the time.

Comment (tdfischer) [inline lib/config.js:10]: That, and to give the configuration some logical structure/namespacing.

**Comment by wilsonianb** [comment]
LGTM :+1:

**Comment by stevenzeiler** [comment]
+1

**Pull request merged**